src/Tizen.NUI.Components/Controls/Notification.cs

```diff
@@ -165,7 +165,7 @@ private Timer Timer
         /// Notification.MakeToast("Hello World!", Notification.ToastBottom).Post(Notification.ToastShort);
```

Comment:
line 163: "period time" -> "period of time"
You can consider changing the sentence a little more: "how to show a toast at the bottom of the screen for a short period of time".

Reply:
Done